portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,9 @@ function addRandomFact() {
   // Display the random fact chosen
   randomFact.classList.add('visible');
 }
+
+async function getMessageFromServer() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  document.getElementById('message-container').innerText = quote;
```

Comment:
since the response is returning HTML, you will want innerHTML instead of innerText.  Otherwise your webpage will print out the html tags as normal text.

Reply:
I was wondering how I could get rid of those tags. Thanks